scripts/qa_crowbarsetup.sh

```diff
@@ -1475,6 +1475,39 @@ function onadmin_repocleanup()
     zypper mr -d sp3sdk
 }
 
+function onadmin_reporeplace()
```

Comment:
The purpose of the function is not clear to me just from this name.  Could you think of a clearer name, or at very least add a comment explaining what it's supposed to do?